doc/ref_cert/RC2/chapters/chapter01.md

```diff
@@ -19,29 +19,34 @@
 
 ## Executive Summary
 
-The Reference Conformance 2 (RC2) was established to ensure implementations of the Reference Architecture 2 (RA2), such as the Reference Implementation 2 (RI2), meet industry driven quality assurance standards for conformance, verification, and validation. For the purpose of this chapter, infrastructure and cloud native network function (CNF) testing will be performed to evaluate **Conformance** (i.e. adherence) to, and demonstrated proficiency with, all aspects of software delivery.  More specifically, Conformance includes:
+The RC2 was established to ensure implementations of the CNTT Reference Architecture 2 (RA2), such as the Reference Implementation 2 (RI2), meet functional and performance requirements derived from RA2 and the CNTT Reference Model (RM). For the purpose of this chapter, cloud infrastructure and cloud native network function (CNF) testing will be specified to evaluate **Conformance** (i.e. adherence) to, and demonstrated proficiency with, all aspects of hardware and software delivery that is within the scope of CNTT. Conformance scope includes:
 
- - Verified implementations of infrastructure and CNF match expected design requirements
- - Clearly stated guidelines for test, badging, and lifecycle management
- - Inclusion of Operational run-books for 3rd party supplier instantiation and validations
- - Evidence, through test results, confirming delivered code matches industrial requirements
- - Interoperability testing with Reference CNFs, ensuring integration stability and life-cycle management of the Reference CNF on the target implementation.
+ - Test criteria with requirements tracebility ensuring implementations of cloud infrastructure match expected capabilites exemplified by CNTT Reference Implementations and CNFs match match expected capabilites for consuming compliant cloud infrastructure resources
```

Comment:
"exemplified" is a "typical example"

Reply:
is there an edit here or just a comment?

Reply:
@xmulligan Yes, as I think "the Reference Implementations" are examples of the "implementations of cloud infrastructure" and not of "expected capabilities".

there is typo: "match match"

Reply:
@xmulligan Please ignore as you have already made changes. Thanks